### PR TITLE
Use hunt instead of hunt_id

### PR DESCRIPTION
**Pull request by supernothing**

### CODE REVIEW

**Review by alanjds — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -20,7 +24,7 @@ default:
 
 dev-pypi:
   tags:
-    - kube
+    - kube-new
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:27]: should be `kube-small-new`

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -35,7 +39,7 @@ dev-pypi:
 
 test-run27:
   tags:
-    - kube
+    - kube-new
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:42]: should be `kube-small-new`

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -49,7 +53,7 @@ test-run27:
 
 test-run35:
   tags:
-    - kube
+    - kube-new
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:56]: should be `kube-small-new`

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -62,7 +66,7 @@ test-run35:
 
 test-run36:
   tags:
-    - kube
+    - kube-new
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:69]: should be `kube-small-new`

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -79,7 +83,7 @@ test-run36:
 
 release-pypi:
   tags:
-    - kube
+    - kube-new
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:86]: should be `kube-small-new`

**Review by mrtizmoatwork — CHANGES_REQUESTED** [review]
use smaller CI runners for basic python tests

File: .gitlab-ci.yml (hunk 1 above)
Comment (supernothing) [reply .gitlab-ci.yml:27]: why? kube-new works, and I assumed these were kube and not kube-small for a reason.

Comment (supernothing) [reply .gitlab-ci.yml:27]: https://gitlab.polyswarm.io/externalci/polyswarm-cli/-/pipelines/34962

**Review by mrtizmoatwork — APPROVED** [review]
ok

**Pull request merged**